scripts/ci-build-kubernetes.sh

```diff
@@ -38,6 +38,7 @@ source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 : "${JOB_NAME:?Environment variable empty or not defined.}"
 
 declare -a BINARIES=("kubeadm" "kubectl" "kubelet")
+declare -a WINDOWS_BINARIES=("kubeadm" "kubectl" "kubelet" "kube-proxy")
```

Comment:
We might also want to check whether these Windows binaries are properly uploaded to the blob storage in `can_reuse_artifacts` function to avoid rebuilding Kubernetes

Reply:
updated